Software/src/Settings.cpp

```diff
@@ -100,46 +100,62 @@ static const QString NumberOfLeds = "Adalight/NumberOfLeds";
 static const QString ColorSequence = "Adalight/ColorSequence";
 static const QString Port = "Adalight/SerialPort";
 static const QString BaudRate = "Adalight/BaudRate";
+static const QString LedMilliAmps = "Adalight/LedMilliAmps";
+static const QString PowerSupplyAmps = "Adalight/PowerSupplyAmps";
 }
 namespace Ardulight
 {
 static const QString NumberOfLeds = "Ardulight/NumberOfLeds";
 static const QString ColorSequence = "Ardulight/ColorSequence";
 static const QString Port = "Ardulight/SerialPort";
 static const QString BaudRate = "Ardulight/BaudRate";
+static const QString LedMilliAmps = "Ardulight/LedMilliAmps";
+static const QString PowerSupplyAmps = "Ardulight/PowerSupplyAmps";
 }
 namespace AlienFx
 {
 static const QString NumberOfLeds = "AlienFx/NumberOfLeds";
+static const QString LedMilliAmps = "AlienFx/LedMilliAmps";
+static const QString PowerSupplyAmps = "AlienFx/PowerSupplyAmps";
```

Comment:
AlienFX has no power supply AFAIK, it's part of the PC. But special-casing it is probably not worth the effort.

Reply:
Good point (unless repurposed!). I'll have to take that into consideration while looking into devices (see below)